bootstrap.yml

```diff
@@ -1498,19 +1542,9 @@ packages:
         quiet: true
 
   - name: python
-    source:
```

Comment:
Hm, don't we want `from_source:` instead? (If it seems to work without that, I'd consider that a bug in xbstrap and not a feature :smiley:)